paper.md

```diff
@@ -35,15 +35,15 @@ This is an early example of a *permutation test*. Like *Monte Carlo tests* [@rob
 
 Scientists and engineers frequently analyze data by considering questions of the following forms.
 
-1. Are the observations sampled from a hypothesized distribution? For instance, in Darwin's experiment, are the differences in heights between paired plants normally distributed?
+1. Are the observations sampled from a hypothesized distribution? For instance, in Darwin's experiment [@darwin1877effects], are the differences in heights between paired plants normally distributed?
```

Comment:
@SaranjeetKaur, you suggested

> it might be helpful to add link to the references in the section "Statement of need" (that is, for all the 3 questions, when you are sharing examples which were introduced in the Summary)

Is this what you had in mind - to cite the existing reference about Darwin's experiment?
Note that Darwin was most interested in #2, but the intent here is to use this situation as an example of when a researcher may be interested in related questions.

Reply:
Yes - for all the 3 questions in the 'Statement of need' it would be nice to have the examples linked.

Reply:
If there were three examples - one per question - I would link them.

As it is, there is only one example: Darwin's experiment.

The reference a actually studied a question like the second question; it did not actually study the first and third questions. I'm suggesting questions that _could have_ been asked about that same data.

I originally considered including three separate, real examples, but I decided against it because would have to describe each one. I thought that would distract from the main ideas.

Reply:
All right, makes sense! Would it then help to link it in the opening statement of that section? Something like:

_Scientists and engineers frequently analyze data by considering questions of the following forms (examples are from the Darwin's experiment [@darwin1877effects]).

1. Are the observations sampled from a hypothesized distribution? For instance, are the differences in heights between paired plants normally distributed?
2. Are the samples drawn from the *same* distribution? For example, does the fertilization method have no effect on plant height?
3. What can be inferred from the samples about the true value of a population statistic? For instance, what is the uncertainty in the true mean difference in heights between paired plants?_